GUI/GUIController.java

```diff
@@ -0,0 +1,44 @@
+package GUI;
```

Comment:
Bring a better implementation for this system?

Reply:
OUI CHEF